packages/nocodb-sdk/src/lib/audit/v1.ts

```
@@ -0,0 +1,884 @@
+  user_id: string;
+  user_email: string;
+  user_name?: string;
+}
+
+export interface UserPasswordChangePayload {}
+
+export interface UserPasswordResetPayload {}
+
+export interface UserPasswordForgotPayload {}
+
+export interface UserEmailVerifyPayload {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)